ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -647,13 +647,15 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-winc
-- as: aws-ipi-private-fips-f28-disasterrecovery
-  cron: 11 11 20 * *
+- as: aws-ipi-private-fips-f1-disasterrecovery
```

Comment:
I suggest `f4` to watch it for a week to check its pass ratio, `f1` is a bit terrible for me.

Reply:
Updated.